packages/python-packages/api-stub-generator/setup.py

```diff
@@ -29,7 +29,10 @@
     author_email="azuresdkengsysadmins@microsoft.com",
     license="MIT License",
     packages=find_packages(),
-    install_requires=["astroid"],
+    install_requires=[
+        "astroid",
+        "charset-normalizer"
```

Comment:
where do we use this new dependency?

Reply:
It's not a new dependency. It was already used throughout APIView. Why it has to be added here I have no idea. But if I remove it, the CI won't pass.